plugins/modules/katello_upload.py

```diff
@@ -64,39 +64,19 @@
     organization: "Default Organization"
 '''
 
-RETURN = '''# '''
+RETURN = ''' # '''
 
-try:
-    from ansible.module_utils.ansible_nailgun_cement import (
-        find_organization,
-        find_product,
-        find_repository,
-        find_package,
-        find_file,
-        ForemanAnsibleModule,
-    )
-
-    from nailgun.entities import (
-        ContentUpload,
-    )
-    from subprocess import check_output
-    import os
-    import hashlib
-except ImportError:
-    pass
+from subprocess import check_output
+import os
+import hashlib
 
-from ansible.module_utils._text import to_native
+from ansible.module_utils.foreman_helper import ForemanApypieAnsibleModule
 
-
-def upload(module, src, repository):
-    content_upload = ContentUpload(repository=repository)
-    if not module.check_mode:
-        content_upload.upload(src)
-    return True
+CONTENT_CHUNK_SIZE = 2 * 1024 * 1024
 
 
 def main():
-    module = ForemanAnsibleModule(
+    module = ForemanApypieAnsibleModule(
```

Comment:
Why not KatelloApypieAnsibleModule?

Reply:
Because it doesn't exist and @evgeni is a lazy bum

Reply:
it is now.